tools/fixup-wheels.sh

```diff
@@ -117,9 +117,10 @@ fixup_wheel () {
         fixup_wheel_macosx
     fi
 
-    zip -r $fixed_whl .
-
     popd # tmpdir
+
+    python -c "from wheel.wheelfile import WheelFile; w = WheelFile('$fixed_whl', mode='w'); w.write_files('$tmpdir'); w.close()"
```

Comment:
Can we move this little scriptlet to its own file in this directory to avoid quoting problems if $fixed_whl or $tmpdir contains special character?

Reply:
Fixed.